examples/receiver-init-poll/main.go

```diff
@@ -5,16 +5,16 @@ import (
 	"time"
 
 	"github.com/sgnl-ai/caep.dev-receiver/pkg"
-	events "github.com/sgnl-ai/caep.dev-receiver/pkg/ssf_events"
+	ssf_events "github.com/sgnl-ai/caep.dev-receiver/pkg/ssf_events"
 )
 
 func main() {
 	// Configure the receiver (specify the poll callback function and poll interval to start polling in each interval)
 	receiverConfig := pkg.ReceiverConfig{
 		TransmitterUrl:     "https://ssf.stg.caep.dev",
-		TransmitterPollUrl: "https://ssf.stg.caep.dev/ssf/streams/poll",
-		EventsRequested:    []events.EventType{0},
-		AuthorizationToken: "f843a2ce-4e94-48d4-aed6-c1617024b245",
+		TransmitterPollUrl: "https://caep-ssf.sbx.sgnl.host/ssf/streams/poll",
+		EventsRequested:    []ssf_events.EventType{0, 1, 2, 3, 4},
+		AuthorizationToken: "affcbce2-4e84-4ec1-8a64-174f05eaca67",
```

Comment:
Do we need a token value to be checked in? Can it just be placeholder?

Reply:
Yes sure, will replace it